src/Catel.Core/Catel.Core.Shared/Logging/Listeners/SeqLogListener.cs

```diff
@@ -89,46 +85,43 @@ protected override string FormatLogEvent(ILog log, string message, LogEvent logE
                 var textWriter = new StringWriter();
                 textWriter.Write("{\"events\":[");
 
-                var logEntries = batchEntries.Select(
-                    batchEntry => FormatLogEvent(batchEntry.Log, batchEntry.Message, batchEntry.LogEvent, batchEntry.ExtraData, batchEntry.Data, FastDateTime.Now))
-                    .Aggregate((log1, log2) => string.Format("{0},{1}", log1, log2));
+                var logEntries = batchEntries.Select(batchEntry => FormatLogEvent(batchEntry.Log, batchEntry.Message, batchEntry.LogEvent, batchEntry.ExtraData, batchEntry.Data, FastDateTime.Now)).Aggregate((log1, log2) => string.Format("{0},{1}", log1, log2));
 
                 textWriter.Write(logEntries);
                 textWriter.Write("]}");
 
                 var message = textWriter.ToString();
 
-                lock (_lock)
+                var baseUri = ServerUrl;
+                if (!string.IsNullOrWhiteSpace(baseUri))
                 {
-                    if (_webClient == null)
+                    using (var webClient = new WebClient { Encoding = Encoding.UTF8 })
                     {
-                        var baseUri = ServerUrl;
                         if (!baseUri.EndsWith("/"))
                         {
                             baseUri += "/";
                         }
 
-                        _webApiUrl = string.Format("{0}{1}", baseUri, BulkUploadResource);
+                        var webApiUrl = string.Format("{0}{1}", baseUri, BulkUploadResource);
 
-                        _webClient = new WebClient {Encoding = Encoding.UTF8};
-                        _webClient.Headers[HttpRequestHeader.ContentType] = "application/json";
+                        webClient.Headers[HttpRequestHeader.ContentType] = "application/json";
 
                         if (!string.IsNullOrWhiteSpace(ApiKey))
                         {
-                            _webClient.Headers.Add(ApiKeyHeaderName, ApiKey);
+                            webClient.Headers.Add(ApiKeyHeaderName, ApiKey);
```

Comment:
We should have a separate method:

CreateWebClient()

this creates a webclient (if needed). Then it adds the default ApiKey as well, so we don't have to do that manually on each call.

Reply:
Fixed